userguide/text_classifier/README.md

```diff
@@ -25,35 +25,53 @@ sentiment for each aspect. For example, when studying reviews about
 mobile phones you may be interested in how people feel about aspects
 such as battery life, screen resolution, size, etc.
 
+#### Loading Data
+
+The [Kaggle SMS Spam Collection Dataset](https://www.kaggle.com/uciml/sms-spam-collection-dataset) provides more than five thousand labeled text messages.[<sup>1</sup>](../datasets.md) After downloading and decompressing the dataset, move the `spam.csv` file to your project directory.
+
+
 #### Introductory Example
 
 Let's now take a look at a simple example of sentiment analysis where
 the task is to predict whether it contains positive or negative
 sentiment.  For instance, the model will predict "positive sentiment"
-for a snippet of text -- whether it is a movie review or a tweet -- when
+for a snippet of text -- whether it is a text message, movie review or a tweet -- when
 it contains words like "awesome" and "fantastic". Likewise, having many
 words with a negative connotation will yield a prediction of "negative
 sentiment".
 
 ```python
 import turicreate as tc
 
-# Load data
-data = tc.SFrame('ratings_data.csv')
+# Only load the first two columns from csv
+# 60 lines won't be parsed because of the bad formatting which you can ignore
+data = tc.SFrame.read_csv('spam.csv', usecols=['v1', 'v2'], column_type_hints=[str, str])
+
+# Rename the columns
+data = data.rename({'v1': 'label', 'v2': 'text'})
```

Comment:
The new data format doesn't have a header. So this should be `data = data.rename({'X1': 'label', 'X2': 'text'})`.